AUTHORS.rst

```diff
@@ -0,0 +1,7 @@
+Authors
+=======
+
+* Diego Rodriguez Rodriguez <diego.rodriguez.rodriguez@cern.ch>
```

Comment:
I have changed the mail alias to `diego.rodriguez@cern.ch` and my name to Diego Rodriguez so it would be:
`* Diego Rodriguez <diego.rodriguez@cern.ch>`
I have to change it in `reanahub/reana` also.